doc/source/rllib/rllib-checkpoints-and-exports.rst

```diff
@@ -0,0 +1,338 @@
+.. _checkpoints-and-exports-docs:
+
+#################
+Using Checkpoints
+#################
+
+
+What's a checkpoint?
+====================
+
+A checkpoint is a set of information, located inside a directory (which may contain
```

Comment:
Maybe first say what it does ("You can use checkpoints to store and load your algorithms") before describing what it technically is?

Reply:
Added a simpler intro paragraph and changed title.